src/main/java/com/sonymobile/jenkins/plugins/kerberossso/PluginImpl.java

```diff
@@ -127,6 +135,15 @@ private void removeFilter() throws ServletException {
     }
 
     /**
+     * Create and attach the filter.
+     * @throws ServletException Unable to add filter.
+     */
+    private void registerFilter() throws ServletException {
+        this.filter = new KerberosSSOFilter(createConfigMap(), new SpnegoKerberosAuthenticationFactory());
+        PluginServletFilter.addFilter(filter);
```

Comment:
This particular refactorization just replaced a verbatim copy of the same code present on both places already. I am quite confident about this part of the patch.